maas-pro.md

```diff
@@ -815,17 +815,22 @@ GET https://partners.voiapp.io/v1/pricing/vehicle/{id}
 
 All successful vehicle interactions responds with the following vehicle model.
 
-| field        | type    | description                                                                                    | presence |
-| ------------ | ------- | ---------------------------------------------------------------------------------------------- | -------- |
-| id           | string  | The vehicle's id (UUID version 4)                                                              | required |
-| type         | string  | For vehicles the type will always be "vehicle"                                                 | required |
-| vehicle_type | string  | The vehicles type , values are `scooter`, `bicycle`                                            | required |
-| batteryLevel | integer | The state of charge of the vehicles' battery in percent (0-100)                                | required |
-| location     | object  | The vehicle’s location                                                                         | required |
-| longitude    | number  | The longitude component of the location                                                        | required |
-| latitude     | number  | The latitude component of the location                                                         | required |
-| code         | string  | The 4-letter alphanumeric vehicle code, also available on the physical vehicle and as QR-code. | required |
-| locked       | bool    | The vehicle lock state                                                                         | required |
+| field          | type    | description                                                                                    | presence |
+| ------------   | ------- | ---------------------------------------------------------------------------------------------- | -------- |
+| id             | string  | The vehicle's id (UUID version 4)                                                              | required |
+| type           | string  | For vehicles the type will always be "vehicle"                                                 | required |
+| vehicle_type   | string  | The vehicles type , values are `scooter`, `bicycle`                                            | required |
+| batteryLevel   | integer | The state of charge of the vehicles' battery in percent (0-100)                                | required |
+| location       | object  | The vehicle’s location                                                                         | required |
```

Comment:
For `object` types would be nice to put references to actual definitions of objects to help readers to understand what is inside

Reply:
just following what we have. I think example is representative enough